test/e2e/util.go

```diff
@@ -46,7 +46,7 @@ import (
 	kbapi "github.com/kubernetes-sigs/kube-batch/pkg/scheduler/api"
 )
 
-var oneMinute = 1 * time.Minute
+var oneMinute = 5 * time.Minute
```

Comment:
If we need more time, we also need change the name `oneMinute`.

Reply:
Addressed this review comment.
Added a new variable and also added a two new helper methods with timeout.